.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        container: ["registry.centos.org/centos/centos:7", "registry.fedoraproject.org/fedora:latest", "registry.fedoraproject.org/fedora:rawhide"]
+        container: ["registry.centos.org/centos/centos:7", "registry.fedoraproject.org/fedora:latest"]
```

Comment:
Why is the rawhide test disabled?

Reply:
whoops. sorry i disabled this and then forgot to investigate further.

it is happening against master too:

https://github.com/fedora-infra/mirrormanager2/runs/4442463827?check_suite_focus=true

not sure how python packaging works with this, but python3-flask is epel only, but python-flask is not. Not sure why it fails in rawhide, but works for latest/f35

 https://src.fedoraproject.org/rpms/python-flask
https://src.fedoraproject.org/rpms/python3-flask

really nore sure how to resolve this

Reply:
reenabled the rawhide test